src/main/java/com/example/carsharing/util/StripeUtil.java

```diff
@@ -0,0 +1,54 @@
+package com.example.carsharing.util;
+
+import com.stripe.Stripe;
+import com.stripe.exception.StripeException;
+import com.stripe.model.checkout.Session;
+import com.stripe.param.checkout.SessionCreateParams;
+import java.math.BigDecimal;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
+public class StripeUtil {
+    private static final String DEFAULT_CURRENCY = "usd";
+    @Value("${stripe.success.link}")
+    private String successUrl;
+    @Value("${stripe.cancel.link}")
+    private String cancelUrl;
+
+    public StripeUtil(@Value("${stripe.api.key}") String apiKey) {
+        Stripe.apiKey = apiKey;
+    }
+
+    public Session createStripeSession(BigDecimal amount, String name) throws StripeException {
+        SessionCreateParams params = SessionCreateParams.builder()
+                .setSuccessUrl(successUrl)
+                .setCancelUrl(cancelUrl)
+                .setMode(SessionCreateParams.Mode.PAYMENT)
+                .addLineItem(
+                        SessionCreateParams.LineItem.builder()
+                                .setQuantity(1L)
```

Comment:
DEFAULT_QUANTITY

Reply:
![image](https://github.com/RomanMenzheres/Car-Sharing/assets/118287818/950db4dd-9404-420f-8ea3-47615cf414a7)
And if it POST it doesn't work